testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/concurrency/ConcurrencyTest.java

```diff
@@ -163,11 +164,15 @@ public void run(int threadIndex, Keycloak keycloak, RealmResource realm) throws
 
             c = realm.clients().get(id).toRepresentation();
             assertNotNull(c);
-            assertTrue("Client " + name + " not found in client list",
-              realm.clients().findAll().stream()
-                .map(ClientRepresentation::getClientId)
-                .filter(Objects::nonNull)
-                .anyMatch(name::equals));
+
+            int findAttempts = 5;
```

Comment:
Maybe we can add more attempts only for H2? Is that possible? This way the other databases run normally as it is now.

Reply:
Done!